models/streamitem_test.go

```diff
@@ -0,0 +1,146 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/bwmarrin/snowflake"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStreamItemStore(t *testing.T) {
+	tx := TestDB.Begin()
+	defer tx.Rollback()
+
+	store := NewStreamItemStore(tx)
+
+	// Empty DB - everything should be not found
+
+	t.Run("Not Found", func(t *testing.T) {
+		_, err := store.GetItem(snowflake.ID(353894652568535040))
+		require.EqualError(t, err, "record not found")
+	})
+
+	t.Run("Empty", func(t *testing.T) {
+		l, err := store.GetItems(snowflake.ID(353894652568535040), Beginning, After, 20)
+		require.NoError(t, err, "GetItems")
+		require.Equal(t, 0, len(l))
+	})
+
+	t.Run("Not Found by Entity ID", func(t *testing.T) {
+		_, err := store.GetItemByEntityID(snowflake.ID(353894652568535040), snowflake.ID(384408932061417472))
+		require.EqualError(t, err, "record not found")
+	})
+
+	// Now we should insert some items into a stream
+	t.Run("Insert", func(t *testing.T) {
+		t.Run("Candles", func(t *testing.T) {
+			b, i, err := store.TryInsertItem(snowflake.ID(353894652568535040), snowflake.ID(384408932061417472))
```

Comment:
`b` is a strange variable name, could it be something like `item` and `inserted`?